Sources/SnowManLogic/GameLogic/Game.swift

```diff
@@ -0,0 +1,29 @@
+public class Game {
+	var word: String
+	var guessManager: GuessManager
+	var view: Viewable
+
+	public init(word: String, guessManager: GuessManager, view: Viewable) {
+		self.word = word
+		self.guessManager = guessManager
+		self.view = view
+	}
+
+	public func checkForGuessInWord(guess: Character) -> Bool {
+		return word.lowercaseString.characters.contains(guess)
+	}
+
+	public func isWinner() -> Bool {
+		let letters = separateLetters()
+		let unguessedLetters = guessManager.findUnguessedLetters(letters)
+		return unguessedLetters.isEmpty
+	}
+
+	public func gameOver() -> Bool {
```

Comment:
Should use a consistent naming convention for your methods that return a boolean value.